themes/finna2/templates/content/register_details_en-gb.phtml

```diff
@@ -4,86 +4,78 @@ $this->content()->setHeading($this->translate('Register Details')); ?>
-  <p> In addition, users’ personal data are processed in an analytics system (Matomo) to monitor the use of services if users have given their consent. The analytics system used in our usage analyses is configured to anonymize the user's IP address by resetting the last two octets of the IP address to zero. We use the analytical system on the servers of CSC - IT Centre for Science so that tracking data cannot be leaked out of this safe environment.</p>
+  <p>In addition, users’ personal data are processed in an analytics system (Matomo) to monitor the use of services if users have given their consent. The analytics system used in our usage analyses is configured to anonymize the user's IP address by resetting the last two octets of the IP address to zero. We use the analytical system on the servers of CSC - IT Centre for Science so that tracking data cannot be leaked out of this safe environment.</p>
   <p>When logging onto this online service, information about the event (e.g. the user’s IP address) is stored temporarily for processing.</p>
   <p>Grounds for processing of personal data include the controller’s legitimate interests and data subjects’ consent (personal data processed in the analytics system).</p>
 
   <h3>5. Register contents</h3>
   <h4>Basic user information</h4>
   <ul class="list-bullets">
-      <li>Username</li>
-      <li>Language</li>
-      <li>Name</li>
-      <li>Relationship to educational institution or other organisation/li>
-      <li>Email address</li>
-      <li>Library card information</li>
+    <li>Username</li>
+    <li>Language</li>
+    <li>Name</li>
+    <li>Relationship to educational institution or other organisation/li>
+    <li>Email address</li>
+    <li>Library card information</li>
   </ul>
   <h4>Information collected for personalised services</h4>
   <ul class="list-bullets">
-      <li>Personal settings</li>
-      <li>Favouritese</li>
-      <li>Tags</li>
-      <li>Comments</li>
-      <li>Reviews</li>
-      <li>Reservations</li>
-      <li>Loans</li>
-      <li>Fees</li>
-      <li>References added to bookshelf</li>
-      <li>Materials added to My materials</li>
-      <li>Journal references added to My journals</li>
+    <li>Personal settings</li>
+    <li>Favourites</li>
+    <li>Tags</li>
+    <li>Comments</li>
+    <li>Reviews</li>
+    <li>Holds</li>
+    <li>Checked out items</li>
+    <li>Fines</li>
+    <li>Saved searches</li>
+    <li>Materials added to My materials</li>
+    <li>Journal references added to My journals</li>
```

Comment:
Näissä vielä termit hukassa. Ehkä riittäisi yksi rivi, jossa "Materials added to Saved Items", "Suosikkeihin lisätyt aineistot", "Material som sparats i favoriter".